### PR TITLE
Fix rendererButton size to 44x44

### PR DESCRIPTION
**Pull request by Mikanbu**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md) and run `bundle exec fastlane lint` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

File: Sources/VLCLibraryViewController.m [hunk 1]
```diff
@@ -414,10 +417,14 @@ - (void)_displayEmptyLibraryViewIfNeeded
...
-        self.navigationItem.rightBarButtonItems = @[self.editButtonItem, _rendererBarButton];
+
+        UIBarButtonItem *fixedSpace = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemFixedSpace target:nil action:nil];
+        fixedSpace.width = NAVBAR_FIXED_SPACE;
+        self.navigationItem.rightBarButtonItems = @[self.editButtonItem, fixedSpace, _rendererBarButton];
         if (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad) {
+            fixedSpace.width = NAVBAR_FIXED_SPACE_IPAD;
             self.displayModeBarButtonItem = [[UIBarButtonItem alloc] initWithImage:[UIImage imageNamed:@"tableViewIcon"] style:UIBarButtonItemStylePlain target:self action:@selector(toggleDisplayedView:)];
-            self.navigationItem.rightBarButtonItems = @[self.displayModeBarButtonItem, self.editButtonItem, _rendererBarButton];
+            self.navigationItem.rightBarButtonItems = @[self.displayModeBarButtonItem, self.editButtonItem, fixedSpace, _rendererBarButton];
```
Comment (carolanitz) [inline Sources/VLCLibraryViewController.m]: how about using a stackview here btw ?

Comment (Mikanbu) [reply Sources/VLCLibraryViewController.m]: Might be missing something but I don't think I can assign a stackView to an array here?

Comment (carolanitz) [reply Sources/VLCLibraryViewController.m]: you can put a stackview in an array though

Comment (Mikanbu) [reply Sources/VLCLibraryViewController.m]: I don’t see why.
Doesn’t it get handled by a stackview internally anyway?

Comment (carolanitz) [reply Sources/VLCLibraryViewController.m]: I dislike the fixedspace. I'd rather a bigger button and a stackview would take care of that by scaling them all to the same size. That would increase the hittable area

**Comment by carolanitz** [comment]
@Mikanbu I just understood why there is no padding in the first place. The renderer button is too small. Buttons by Apples UX guideline should have roughly 44x44 size, the renderer button is far from it. here is the reference (https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/adaptivity-and-layout/) if you fix that you won't need the fixed space. I believe that is an issue in 3.0 and master

**Comment by Mikanbu** [comment]
Updated!

**Pull request merged**